### PR TITLE
feat: Generate manifests from Github actions.

### PR DESCRIPTION
**Pull request by gary-van-woerkens**

### CODE REVIEW

File: src/environments/index.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+export const getProject = (): string | undefined => {
+  // TODO: Use of "toLowerCase" only to match snapshots, to be removed.
+  return (
+    process.env.CI_PROJECT_PATH?.toLowerCase() ?? process.env.GITHUB_REPOSITORY
+  );
+};
+
+export const isProduction = (): boolean =>
+  process.env.PRODUCTION || process.env.SOCIALGOUV_PRODUCTION ? true : false;
```
Comment (revolunet) [inline src/environments/index.ts:41]:
```suggestion
  !!(process.env.PRODUCTION || process.env.SOCIALGOUV_PRODUCTION);
```

File: src/utils/createDeployment.ts [hunk 2]
```diff
@@ -38,11 +40,16 @@ export interface DeploymentParams {
...
+  // const tag = process.env.CI_COMMIT_TAG
+  //   ? process.env.CI_COMMIT_TAG.slice(1)
+  //   : process.env.CI_COMMIT_SHA;
+  // const image =
+  //   params.image || `${process.env.CI_REGISTRY_IMAGE}/${params.name}:${tag}`;
+  // console.log("process.env.CI_PROJECT_NAME", process.env.CI_PROJECT_NAME);
+
   const image =
-    params.image || `${process.env.CI_REGISTRY_IMAGE}/${params.name}:${tag}`;
+    params.image || getGitlabRegistryImagePath({ name: params.name });
```
Comment (revolunet) [inline src/utils/createDeployment.ts:51]: we should default to ghcr here

File: src/components/namespace/index-github.test.ts [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  __esModule: true,
+  default: () => githubMock,
+};
+
+beforeEach(() => {
+  jest.resetModules();
+});
+
+test("should create a namespace", async () => {
```
Comment (revolunet) [inline src/components/namespace/index-github.test.ts:19]:
```suggestion
test("should create a github namespace", async () => {
```

**Comment by revolunet** [comment]
i dont think we should have the gitlab annotations in the github environment :
`   annotations: {      "app.gitlab.com/app": "socialgouv-sample",      "app.gitlab.com/env": "my-test",}`

File: src/components/app/__snapshots__/index-github.test.ts.snap [hunk 4]
```diff
@@ -0,0 +1,919 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```
Comment (revolunet) [inline src/components/app/__snapshots__/index-github.test.ts.snap:1]: les urls des ingress sont incorrectes ici

**Comment by douglasduteil** [comment]
Replaced by  https://github.com/SocialGouv/kosko-charts/pull/533

**Pull request closed without merging**